build.gradle

```
@@ -11,10 +11,10 @@ repositories {
 }
 
 dependencies {
-    compileOnly 'javax.servlet:javax.servlet-api:4.0.1'
-    implementation 'org.springframework:spring-webmvc:5.3.15'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test:2.6.3'
+    implementation 'org.springframework.boot:spring-boot-starter-web:2.6.3'
+    developmentOnly("org.springframework.boot:spring-boot-devtools")
```

요거는 왜 넣었을까요?

---

위에는 그루비 스타일로 작성해주셨고, 여기는 평범한 메소드 스타일로 작성해 주셨네요. 별거 아니니까 통일해보쉴?

```suggestion
    developmentOnly 'org.springframework.boot:spring-boot-devtools'
```

---

`devtools`는 `livereload` 적용해서 코드 수정이 바로 반영됐는지 확인해보려구 넣었습니다!

---

`Groovy` 문법은 이해가 부족해서 한 번 찾아보도록 하겠습니다 :)

---

그 짧은 틈에 거기까지..!